requirements.txt

```diff
@@ -2,3 +2,11 @@ alembic==1.3.0
 canonicalwebteam.flask-base==0.3.3
 psycopg2==2.8.4
 sqlalchemy==1.3.10
+Wand==0.5.8
+more-itertools==3.2.0
+pilbox==1.3.4
+scour==0.37
+sh==1.12.14
```

Comment:
Why do we need all these dependencies? Are you really using them all directly?

Reply:
We import all these dependencies on our code.

Reply:
Ah so they were needed before, just not mentioned. Christ.

Reply:
They are on master, but when I created the assets branch I started the requirements file from scratch so we can add as we go.